security-framework/src/os/macos/certificate.rs

```diff
@@ -30,6 +32,9 @@ pub trait SecCertificateExt {
     /// subset.
     fn properties(&self, keys: Option<&[CertificateOid]>)
         -> Result<CertificateProperties, CFError>;
+
+    /// Returns the SHA-256 fingerprint of the certificate.
+    fn fingerprint(&self) -> Result<String, CFError>;
```

Comment:
I think it'd be fine to return raw bytes, and leave hex-encoding to callers.

Reply:
Done.